testdata/multiple_level_doc/TEST-100-ORD.md

```diff
@@ -0,0 +1,70 @@
+# ReqTraq Test File
+
+This file is used as a test input for the reqtraq tool.
+
+## Valid Requirements
+
+### REQ-TEST-SYS-1 VALID System-level requirement, no parent
+
+Must be linked from component requirements in this document.
+
+###### Attributes:
+- Parents:
+- Rationale: Rationale
+- Component Allocation: System
+
+### REQ-TEST-SYS-2 VALID System-level requirement, customer parent
+
+Must be linked from component requirements in this document.
+
+###### Attributes:
+- Parents: REQ-TEST-CST-1
+- Rationale:
+- Component Allocation: System
+
+### REQ-TEST-SYS-3 VALID Component-level requirement, no parent
+
+Component1. Must be linked from high-level requirement.
+
+###### Attributes:
+- Parents:
+- Rationale: Rationale
+- Component Allocation: Component1
+
+### REQ-TEST-SYS-4 VALID Component-level requirement, system parent
+
+Component1. Must be linked from high-level requirement.
+
+###### Attributes:
+- Parents: REQ-TEST-SYS-1
+- Rationale:
+- Component Allocation: Component1
```

Comment:
Aha! So the plan is to add these to our high-level requirements to make sure that one item can only link to its own item reqs? That sounds nice actually